snooty/rstspec.toml

```diff
@@ -1468,6 +1468,9 @@ type = {link = "https://docs.microsoft.com/en-us/xamarin/%s"}
 [rstobject."py:meth"]
 type = "callable"
 
+[rstobject."py:method"]
```

Comment:
Not that I'm opposed, but I'd like a little context: is both this *and* the above necessary?

Reply:
I mean, no. I kind of want to force them to do py:method bu realistically that's probably not going to happen, so I pulled this out :D